AutomatedTesting/Gem/PythonTests/scripting/TestSuite_Periodic.py

```diff
@@ -217,12 +217,26 @@ class TestScriptCanvasTests(object):
     """
     The following tests use hydra_test_utils.py to launch the editor and validate the results.
     """
+    def test_NodePalette_HappyPath_CanSelectNode(self, request, editor, launcher_platform):
+        expected_lines = [
+            "Success: Category can be selected",
+            "Success: Node can be selected"
+        ]
+        hydra.launch_and_validate_results(
+            request,
+            TEST_DIRECTORY,
+            editor,
+            "NodePalette_HappyPath_CanSelectNode.py",
+            expected_lines,
+            auto_test_mode=False,
+            timeout=60,
+        )
```

Comment:
You implemented Report.result in the EPB hydra script why not make this transition all the way?
```suggestion
from ly_test_tools.o3de.editor_test import EditorBatchedTest, EditorTestSuite

class TestScriptCanvasNodes(EditorTestSuite):

    class test_NodePalette_HappyPath_CanSelectNode(EditorBatchedTest):
        from scripting import NodePalette_HappyPath_CanSelectNode as test_module
```
there's probably a little more to it, but you are so close to updating the test all the way.
keeping in mind that launch_and_validate_results may not be around forever.

Reply:
I'm not sure I fully understand this suggestion. It looks like you are saying to remove the expected_lines since Report.result already does a check.

Reply:
the expected lines is part of hydra.launch_and_validate_results which is a log monitoring read lines for outcome model of testing. this is old and prone to file system issues.
the ly_test_tools.o3de.editor_test is the more current pattern of tests which uses the Report.result to communicate outcome without the log lines.
the pytest script uses the EditorBatchedTest to signal run time collection of tests.

the suggestion is more an all or nothing update the test to the new pattern since you are half way there with the Report.result change.

It's not a required change.

Reply:
@smurly we've add this todo item to one of our backlog tasks that will be taken on after all of the broken scripting tests are fixed up and another test suite conversion that @Broganab is working on is submitted. Doing this prematurely can make a merge-conflict nightmare otherwise.